chart/templates/deployment.yaml

```diff
@@ -167,7 +169,7 @@ spec:
           secretName: tls-ca
 {{- end }}
 {{- if gt (int .Values.auditLog.level) 0 }}
-  {{- if eq .Values.auditLog.destination "hostPath" }}
+  {{- if .Values.auditLog.hostPath }}
```

Comment:
Please just keep `if eq .Values.auditLog.destination "hostPath" `. We don't need to change this line.

there is a default value of `.Values.auditLog.hostPath`. If you do so, no matter if user is using `hostPath` mode, Rancher will write audit logs to the default hostPath on the node.

Reply:
Done